boards/arm/mps/mps2-an500/configs/knsh/defconfig

```diff
@@ -61,7 +61,6 @@ CONFIG_READLINE_CMD_HISTORY=y
 CONFIG_RR_INTERVAL=200
 CONFIG_SCHED_HPWORK=y
 CONFIG_SCHED_HPWORKPRIORITY=192
-CONFIG_SPINLOCK=y
```

Comment:
@crafcat7 please look at this problem.

Reply:
So, Let's merge this commit first ? which does not conflict with the current feature

Reply:
Ok.